bool cutter_left
bool cutter_right
int8 LEDstatus
int8 temp1
int8 temp2
int16 voltage
int16 current 

